target_goal:
  pick: {pose: [1.67495292067, 0.311393855561, 0.0], orientation: [0.0, 0.0, -0.0348673749669, 0.999391948218]}
  place: {pose: [-0.12828520542, 0.344357713297, 0.0], orientation: [0.0, 0.0, 0.999974910843, 0.00708362083333]}
  standby: {pose: [0.420522442548, 0.334483644462, 0.0], orientation: [0.0, 0.0, 0.0925239175095, 0.995710462277]}

arm_pose:
  box_0: [4.684074784307185, 1.5255659977893177, 3.6054583238342683, 0.6911503571581435, 1.8591500938232262, 4.339436439220576, 6.914589908968729]
  box_1: [5.304266945274876, 1.280371019912649, -3.4845649250984487, 1.9477701080469754, 2.1931843972394125, 5.048818664043347, 8.1586280879197]
  box_2: [4.424292208401553, 2.030044475183338, -2.0623279797075513, 1.2911854513091883, 1.3864505850849032, 3.7906521491933978, 8.326409899755781]
  box_3: [4.849358551040773, 1.3655342510414754, -2.775857278979794, 1.4174286080427594, 1.6646000509224306, 4.380751921423738, 7.477647144542077]
  box_4: [4.404185075855346, 2.065297004848826, -1.9935630298328335, 0.9169860847017268, 1.9064446389897254, 3.7768135652210195, 7.753981532938662]
  box_5: [4.858688136971539, 1.0782076652455894, -2.6741628748584647, 1.2438040870911524, 1.766959712684062, 4.286947930980095, 6.978311897003272]
  box_6: [4.742531702866643, 1.4947976987673053, -2.4206027748398022, 0.7091555898140058, 1.814948810262854, 4.382965807237921, 6.946477001964683]
  standby: [0.4115857540963754, 1.6389069379897823, 4.697630806887906, 2.7695549082601216, 4.439948932450128, -2.4345556084229454]
  pre_place: [4.684074784307185, 1.5255659977893177, 3.6054583238342683, 0.6911503571581435, 1.8591500938232262, 4.339436439220576, 6.914589908968729]
  place: [4.684074784307185, 1.5255659977893177, 3.6054583238342683, 0.6911503571581435, 1.8591500938232262, 4.339436439220576, 6.914589908968729]
  retreat_place: [4.684074784307185, 1.5255659977893177, 3.6054583238342683, 0.6911503571581435, 1.8591500938232262, 4.339436439220576, 6.914589908968729]
